### PR TITLE
Adds workforce optimization demo

### PR DESCRIPTION
**Pull request by AlexandreSajus**
Blocked by: https://github.com/Avaiga/taipy/issues/1636

image
scenario
results

https://workforce-optim.taipy.cloud/scenario
https://github.com/Avaiga/demo-workforce-plan

### CODE REVIEW

**Review by FlorianJacta — APPROVED** [review]
LGTM

**Review by FabienLelaquais — APPROVED** [review]
👍

File: docs/gallery/decision_support/4_workforce_optim/index.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+A simple application where you input the number of staff you need per day. A linear optimizer will then minimize the amount of workers hired to meet the demand according to certain constraints such as employees having to work 5 consecutive days and have 2 days off.
+
+[Try it live](https://workforce-optim.taipy.cloud/){: .tp-btn target='blank' }
+[Get it on GitHub](https://github.com/Avaiga/demo-workforce-plan){: .tp-btn .tp-btn--accent target='blank' }
+
+More information about the problem can be found [here](https://towardsdatascience.com/optimize-workforce-planning-using-linear-programming-with-python-47a0b5f89a6f).
+
+# Understanding the Application
+
+You can create new optimization scenarios, edit the demand for each day and submit the optimization from this interface.
```
Comment (jrobinAV) [inline docs/gallery/decision_support/4_workforce_optim/index.md]:
```suggestion
You can create new optimization scenarios, edit the demand for each day and submit the
optimization from this interface.
```

File: docs/gallery/decision_support/4_workforce_optim/index.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+title: Workforce Optimization
+category: decision_support
+data-keywords: optimization scenario community
+short-description: Optimize staff hiring and scheduling to meet demand.
+img: 4_workforce_optim/images/results.png
+hide:
+    - toc
+---
+
+A simple application where you input the number of staff you need per day. A linear optimizer will then minimize the amount of workers hired to meet the demand according to certain constraints such as employees having to work 5 consecutive days and have 2 days off.
```
Comment (jrobinAV) [inline docs/gallery/decision_support/4_workforce_optim/index.md]:
```suggestion
A simple application where you input the number of staff you need per day. A linear optimizer
will then minimize the amount of workers hired to meet the demand according to certain
constraints such as employees having to work 5 consecutive days and have 2 days off.
```

File: docs/gallery/decision_support/4_workforce_optim/index.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+[Try it live](https://workforce-optim.taipy.cloud/){: .tp-btn target='blank' }
+[Get it on GitHub](https://github.com/Avaiga/demo-workforce-plan){: .tp-btn .tp-btn--accent target='blank' }
+
+More information about the problem can be found [here](https://towardsdatascience.com/optimize-workforce-planning-using-linear-programming-with-python-47a0b5f89a6f).
```
Comment (jrobinAV) [inline docs/gallery/decision_support/4_workforce_optim/index.md]:
```suggestion
More information about the problem can be found
[here](https://towardsdatascience.com/optimize-workforce-planning-using-linear-programming-with-python-47a0b5f89a6f).
```

**Review by jrobinAV — APPROVED** [review]

**Comment by AlexandreSajus** [comment]
Closing for now, might reopen this after demos migration

**Pull request closed without merging**